src/eradiate/pipelines/_gather.py

```diff
@@ -134,6 +127,117 @@ def transform(self, x: dict) -> xr.Dataset:
         result = result.rename({"img": var})
         result[var].attrs.update(var_metadata)
 
-        logger.debug("gather: end")
```

Comment:
Comment: this convinces that we need something to automate such logging operations, it is very useful for debugging.